config/locales/hi.yml

```diff
@@ -0,0 +1,2 @@
+hi:
+  hello: "नमस्कार"
```

Comment:
This file can be added, when we have to provide multilingual support

Reply:
So, should I add it or not?